components/suggestion/emoticon_provider.tsx

```diff
@@ -148,7 +164,7 @@ export default class EmoticonProvider extends Provider {
             }
         }
 
-        const sortEmojisHelper = (a, b) => {
+        const sortEmojisHelper = (a: any, b: any) => {
             return compareEmojis(a, b, partialName);
```

Comment:
It should have been
```
compareEmojis(a.emoji, b.emoji, partialName)
```

But this is separate but to be solved in another pr

Reply:
@M-ZubairAhmed, should these changes be applied here?

Reply:
no need can be done separately